addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9742,9 +9742,10 @@ msgctxt "#19179"
 msgid "Deleted"
 msgstr ""
 
+#. Settings -> Interface -> Other -> Startup window
 #: system/settings/settings.xml
 msgctxt "#19180"
-msgid "TV"
+msgid "TV channels"
```

Comment:
Yep. It's exclusively used for Settings -> Interface -> Other -> Startup window value. Checked xbmc core (incl. Estuary and Estouchy) + Confluence.